application/tests/mock/personnel/Component.php

```diff
@@ -75,4 +75,13 @@ public function createPersonnelUserFromData($data)
         return $user;
     }
 
-}
\ No newline at end of file
+    /*
+     * This method is not used, but added to satisfy the interface.
+     * Also note that this class is not used at present, as tests
+     * are currently be run using an actual broker container. Need
+     * to decide whether to eliminate this class or go back to using it.
+     */
+    public function updateUser($properties)
+    {
```

Comment:
It may be worth "enforcing" the fact that this method isn't used by throwing some kind of exception if it is ever called. Suggestion:
```suggestion
    {
        throw new \Exception(__FUNCTION__ . ' has not been implemented in this class.');
```